external/storm-hdfs/src/test/java/org/apache/storm/hdfs/trident/TridentSequenceTopology.java

```diff
@@ -60,7 +61,7 @@ public static StormTopology buildTopology(String hdfsUrl){
                 .withSequenceFormat(new DefaultSequenceFormat("key", "sentence"))
                 .withRotationPolicy(rotationPolicy)
                 .withFsUrl(hdfsUrl)
-                .withConfigKey("hdfs.config")
```

Comment:
yes，I think we should change to hdfs.config for this reason, I will add a patch later.